Sources/OTT/KalturaOTTPlayer.swift

```diff
@@ -280,10 +283,12 @@ extension KalturaOTTPlayer: EntryLoader {
             return
         }
         
+        ottMediaOptions = mediaOptions
+        
         if let newKS = mediaOptions.ks, !newKS.isEmpty {
             updatePlayerOptionsKS(newKS)
         }
```

Comment:
Now that the ottMediaOptions is set here the update PlayerOptions can be removed because it is done in the didSet.

```suggestion
        ottMediaOptions = mediaOptions
        
```